index.html

```diff
@@ -291,9 +291,9 @@ <h1 class="primary">About Me</h1>
     <h2 class="h2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
     <div class="input">
      <div class="inp ini">
-      <input type="text" id="name" placeholder="Full name" maxlength="30" required name="full name" class="inp1">
-      <input type="text" id="name" placeholder="First name" required maxlength="30" name="firstname" class="inp2">
-      <input type="text" id="name" placeholder="Last name" required maxlength="30" name="lastname" class="inp3">
+      <input type="text" id="name" placeholder="Full name" maxlength="30" name="full name" class="inp1">
+      <input type="text" id="name" placeholder="First name" maxlength="30" name="firstname" class="inp2">
+      <input type="text" id="name" placeholder="Last name" maxlength="30" name="lastname" class="inp3">
```

Comment:
- [ Optional ] Kindly I recommend you  to keep the `required` attribute to all input fields because the form  should not be submitted if one of the input filed is not filled out.